modules/redisearch.ts

```diff
@@ -540,289 +562,441 @@ export class Redisearch extends Module {
- * @param stopwords.stopword The stopword argument of the 'STOPWORDS' parameter.
- */
-export type FTCreateParameters = {
+*/
+export interface FTCreateParameters {
+    /**
+    * The expression of the 'FILTER' parameter. is a filter expression with the full RediSearch aggregation expression language.
+    */
     filter?: string,
+    /**
+    * The field of the 'PAYLOAD' parameter. If set indicates the document field that should be used as a binary safe payload string to the document, that can be evaluated at query time by a custom scoring function, or retrieved to the client.
+    */
     payloadField?: string,
+    /**
+    * The 'MAXTEXTFIELDS' parameter. For efficiency, RediSearch encodes indexes differently if they are created with less than 32 text fields.
+    */
     maxTextFields?: number,
-    noOffsets?: string,
+    /**
+    * The 'NOFFSETS' parameter. If set, we do not store term offsets for documents (saves memory, does not allow exact searches or highlighting).
+    */
+    noOffsets?: boolean,
+    /**
+    * The 'TEMPORARY' parameter. Create a lightweight temporary index which will expire after the specified period of inactivity.
+    */
     temporary?: number,
-    nohl?: string,
-    noFields?: string,
-    noFreqs?: string,
+    /** 
+    * The 'NOHL' parameter. Conserves storage space and memory by disabling highlighting support. If set, we do not store corresponding byte offsets for term positions.
+    */
+    nohl?: boolean,
+    /**
+    *  The 'NOFIELDS' parameter. If set, we do not store field bits for each term.
+    */
+    noFields?: boolean,
+    /**
+    *  The 'NOFREQS' parameter.  If set, we avoid saving the term frequencies in the index.
+    */
+    noFreqs?: boolean,
+    /**
+    *  The 'SKIPINITIALSCAN' parameter. If set, we do not scan and index. 
+    */
     skipInitialScan?: boolean
-    prefix?: {
-        count: number,
-        name: string
-    }[],
+    /**
+    *  The 'PREFIX' parameter. tells the index which keys it should index.
+    */
+    prefix?: string[],
+    /**
+    * The 'LANGUAGE' parameter.  If set indicates the default language for documents in the index.
+    */
     language?: string,
+    /**
+    * The 'LANGUAGE_FIELD' parameter. If set indicates the document field that should be used as the document language.
+    */
     languageField?: string,
+    /**
+    * The 'SCORE' parameter. If set indicates the default score for documents in the index. 
+    */
     score?: string,
+    /**
+    * The 'SCORE_FIELD' parameter. If set indicates the document field that should be used as the document's rank based on the user's ranking. 
+    */
     scoreField?: string
-    stopwords?: {
-        num: number,
-        stopword: string
-    }
+    /**
+    * The 'STOPWORDS' parameter. If set, we set the index with a custom stopword list, to be ignored during indexing and search time. 
+    */
+    stopwords?: string[],
```

Comment:
According to the docs it should be as it was, please verify before making changes.

Reply:
Num is a redundant parameter that I believe is useless in this situation as it should always be the same as the number of stopwords.
Ex:
```js
stopword: {
   num: 4,
   stopword: "this that a an"
},
//This is more elegant and simpler
stopwords: ["this", "that", "a", "an"]
```